### PR TITLE
chore: transpile TS on init not every execution

### PR DESCRIPTION
**Pull request by robert-bo-davis**
## Description

TS transpile is *very* slow inside of virtualized docker (on macos or windows). The Alpine Linux based terraform container runs node 18.x. Node/npm version mismatches between npm install and running scripts from the scripts folder cause issues. This moves `npm install` and `tsx` of the scripts into the init phase so that they are run inside of the container with the correct node versions.

### CODE REVIEW

**Review by stephenhand — APPROVED** [review]
LGTM

File: twilio-iac/scripts/installScripts.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd ../../scripts/
+
+npm i
```
Comment (GPaoloni) [inline twilio-iac/scripts/installScripts.sh:5]: Isn't `npm ci` preferred for this cases? No really something needed, just asking.

**Review by GPaoloni — APPROVED** [review]
Great! From the user's perspective nothing changes, we run the same commands, right?

**Comment by robert-bo-davis** [comment]
> Great! From the user's perspective nothing changes, we run the same commands, right?

Yup!

**Pull request merged**